app/validation.go

```diff
@@ -230,50 +239,6 @@ func checkForDuplicateProperty(prefix, property, value string, light map[string]
 	return allErrs
 }
 
-// checkBOSHDefaults reports all properties which were given differing
-// defaults across BOSH releases and the jobs inside.
-func (f *Fissile) checkBOSHDefaults(pd propertyDefaults) {
```

Comment:
No, as everything moves to per job properties with almost no namespacing, these will only increase in number.
In your example - what was the other job? Did you find it because of fissile validation?
I don't consider that list to be useful anymore, the number of warnings is too large and we can't do anything about it.

Reply:
The other job was in router somewhere, and yes it was found via fissile validation, IIRC. (Sebi was looking at making it customizable)

I'm okay with not caring about it, though! :D